frontend/src/components/header/header.tsx

```diff
@@ -17,29 +32,54 @@ export const Header: React.FC = () => {
     setWishlistCount(wishlist.models.length + wishlist.complectations.length);
   }, [wishlist]);
 
+  const { data } = useGetUserQuery();
+
   const user = {
     favorites: wishlistCount,
     comparisons: 5,
     notifications: 7,
-    avatar: undefined,
+    ...data,
   };
 
   return (
-    <div className={styles.header}>
-      <div className={styles.container}>
-        <Logo />
-        <Navigate />
-      </div>
-      {user ? (
-        <PrivateElements
-          avatar={user.avatar}
-          notifications={user.notifications}
-          comparisons={user.comparisons}
-          favorites={user.favorites}
-        />
-      ) : (
-        <PublicElements />
-      )}
-    </div>
+    <>
+      <AppBar
+        sx={{
+          background: '#ffffff',
+          boxShadow: 0,
+        }}
+      >
+        <Toolbar>
+          <Link to={AppRoute.ROOT}>
+            <img className={styles.logo} src={Logo} alt="Autoline" />
+          </Link>
+          {isMatchSm ? (
+            <DrawerComp />
```

Comment:
What is the `Comp`? Please be more explicit. If it means the `component` than we should restate already known fact

Reply:
We had two headers in the project and I combined them into one
I think it's a component, so I renamed it now)